juniper/src/ast.rs

```diff
@@ -496,45 +541,48 @@ impl<'a> VariableDefinitions<'a> {
 mod tests {
     use super::InputValue;
     use parser::Spanning;
+    use value::DefaultScalarValue;
+
+    type TestValue = InputValue<DefaultScalarValue>;
```

Comment:
Is this needed anymore now that you have a default type?

Reply:
This could be replaced by `InputValue`, but the type annotation will be required in this case. (Because there is no way for rustc to see which type should be used for `S` here)